crates/ruff_python_formatter/src/statement/stmt_delete.rs

```diff
@@ -28,19 +31,15 @@ impl<'a> DeleteList<'a> {
 
 impl Format<PyFormatContext<'_>> for DeleteList<'_> {
     fn fmt(&self, f: &mut Formatter<PyFormatContext<'_>>) -> FormatResult<()> {
-        write!(
-            f,
-            [&format_with(|f| {
-                let separator = text(", "); // TODO(cnpryer)
-                let mut join = f.join_with(&separator);
+        let separator =
+            format_with(|f| group(&format_args![text(","), soft_line_break_or_space(),]).fmt(f));
```

Comment:
Nit
```suggestion
        let separator = group(&format_args![text(","), soft_line_break_or_space()]);
```

Reply:
You probably want to wrap the whole list in a group rather than each separator so that all `soft_line_breaks` become `hard_line_breaks` when a single element doesn't fit on the line. Or you may not need the `group` at all. But that depends on the expected formatting.

Reply:
> let separator = group(&format_args![text(","), soft_line_break_or_space()]);

Hmm. I think I tried this but had temporary value issues. I'll check it out.

> You probably want to wrap the whole list in a group rather than each separator

I play around with it. Thanks!

Reply:
I think https://github.com/astral-sh/ruff/pull/5169#discussion_r1257381587 might resolve this as well https://github.com/astral-sh/ruff/pull/5169#discussion_r1257381587